### PR TITLE
Update instructions to be independent of minikube

### PR DESCRIPTION
**Pull request by alban**
This should work both on Minikube and Gardener.

### CODE REVIEW

File: 03-Snooping_Operations/README.md [hunk 1]
```diff
@@ -36,14 +36,22 @@ website.
 
 ```
 kubectl apply -f nginx.yaml
-minikube service nginx-deployment
 ```
 
-Since we use minikube, we use the "minikube service" command to make the load
+If the cluster provides a load balancer, you can use the following command to
```
Comment (marga-kinvolk) [inline 03-Snooping_Operations/README.md:41]:
```suggestion
If the cluster provides a load balancer, we can use the following command to
```

File: 03-Snooping_Operations/README.md [hunk 2]
```diff
@@ -36,14 +36,22 @@ website.
...
+know the URL of the nginx service:
+```
+kubectl get svc nginx-deployment
+URL=$(kubectl get svc nginx-deployment -o go-template='{{(index .status.loadBalancer.ingress 0).hostname}}')
+curl "$URL/hello.txt"
+```
+
+If the cluster is minikube, we use the "minikube service" command to make the load
```
Comment (marga-kinvolk) [inline 03-Snooping_Operations/README.md:49]:
```suggestion
If the cluster is minikube, we can use the "minikube service" command to make the load
```

**Review by margamanterola — COMMENTED** [review]
LGTM. Just a couple of suggestions to make the language the same for both cases.

**Review by margamanterola — APPROVED** [review]
We use the minikube ssh thing in the 04 as well, but I think it's ok to leave this in 05 for now.

**Pull request merged**